Dockerfile

```diff
@@ -397,7 +400,8 @@ RUN if [ "${PIP_PKGS}" != "" ] || [ "${IMAGE_LAYER_PYTHON}" != "0" ] || [ "${IMA
     ; fi
 
 # Add nodejs app layer
-RUN if ( [ "${NPM_PKGS}" != "" ] || [ "${IMAGE_LAYER_NODEJS}" != "0" ] || [ "${IMAGE_LAYER_NODEJS_EXT}" != "0" ] ) && [ "${ARCH}" != "arm32v5" ]; then \
+RUN if ( [ "${NPM_PKGS}" != "" ] || [ "${IMAGE_LAYER_NODEJS}" != "0" ] || [ "${IMAGE_LAYER_NODEJS_EXT}" != "0" ] ) && ( [ "${ARCH}" == "AMD64" ] || [ "${ARCH}" == "ARMv7" ] || [ "${ARCH}" == "ARM64v8" ]  ||  \
+         [ ${TARGETPLATFORM} == 'linux/amd64' ] || [ ${TARGETPLATFORM} == 'linux/arm/v7' ] || [ ${TARGETPLATFORM} == 'linux/arm64' ] ) ; then \
```

Comment:
It seems new variables TARGETARCH and TARGETPLATFORM are used inconsistantly. This shall be corrected before merging into dev branch.

Reply:
Also make sure the content variable is not tied to be available on Github Actions only as otherwise local builds on personal devices may break.

Reply:
TARGETARCH and TARGETPLATFORM  are standard docker variables, noting special to github actions:

https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

TARGETARCH  isn't used on this line, what is your approach?